ssh-find-agent.sh

```diff
@@ -194,6 +194,11 @@ set_ssh_agent_socket() {
 		export SSH_AUTH_SOCK=$SOCK
 	fi
 
+     # set agent pid
```

Comment:
This script uses TAB indentation - should indent to match.

Reply:
Thank you for pointing that out. I fixed the indentation.